### ISSUE TITLE
Remove `CertMgr.exe` and update `installCert.bat`

### ISSUE THREAD

**Issue by sitiom** [ge9/IddSampleDriver#6]
From https://github.com/roshkins/IddSampleDriver/issues/26#issue-1300134806:
> Please remove `certmgr.exe` next time. You can replace the contents of `installCert.bat` with this:
> 
> ```batchfile
> @echo off
>
> set CERTIFICATE="%~dp0IddSampleDriver.cer"
>
> certutil -addstore -f root %CERTIFICATE%
> certutil -addstore -f TrustedPublisher %CERTIFICATE%
>
> pause
> ```

**Comment by sitiom** [comment]
Works as intended when you Right Click > Run as admin:

![image](https://user-images.githubusercontent.com/56180050/178428722-7bbac99d-2390-46e0-a446-ec2fc03ed1aa.png)

**Comment by song-fangzhen** [comment]
Hi, may I ask a newbie question: how can I generate the `.cer` file requested by the `installCert.bat`?
I try to write a `makeCert.bat` but it doesn't work:
```
.\makecert.exe -m 120 -r -pe -ss PrivateCertStore -n "CN=WDKTestCert" -eku 1.3.6.1.5.5.7.3.3 IddSampleDriver.cer
```
Any suggestions are appreciated!

**Comment by ge9** [comment]
Sorry, I don't understand what "generate the `.cer` file" means.  You should be using the `.cer` file from the release page: https://github.com/ge9/IddSampleDriver/releases

**Comment by song-fangzhen** [comment]
@ge9 Thanks for your reply! I'm trying to make some modifications to the driver [here](https://github.com/microsoft/Windows-driver-samples/tree/main/video/IndirectDisplay) and want to install the modified version. So I think I need a different `.cer` from  https://github.com/ge9/IddSampleDriver/releases.

**Comment by song-fangzhen** [comment]
Never mind, I've found the related [document ](https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/signing-a-driver-during-development-and-testing#sharing-a-signing-certificate)on how to achieve that. Thanks anyway!

**Comment by KINGLIFER** [comment]
is there a way to uninstall this? Please and thank you.

**Comment by ge9** [comment]
I've never tried but https://github.com/roshkins/IddSampleDriver/issues/10 will do the job.
You may also need to remove certificates from the store.

**Comment by ge9** [comment]
fixed in release [0.0.1.3](https://github.com/ge9/IddSampleDriver/releases/tag/0.0.1.3) .
